<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/image/resource/logo32_32.ico</normaloff>:/image/resource/logo32_32.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,7">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="qStartWgt" name="startWgt" native="true"/>
      </item>
      <item>
       <widget class="qShowMvStatus" name="showMVStatusWgt" native="true"/>
      </item>
      <item>
       <widget class="ServoWgt" name="servoWgt" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="CamerWgt" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="displayImageLabel">
         <property name="text">
          <string>WaitForConnect</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuconfig">
    <property name="title">
     <string>config</string>
    </property>
    <addaction name="actioncarmera"/>
   </widget>
   <addaction name="menuconfig"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actioncarmera">
   <property name="text">
    <string>carmera</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>qStartWgt</class>
   <extends>QWidget</extends>
   <header>qstartwgt.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qShowMvStatus</class>
   <extends>QWidget</extends>
   <header>qshowmvstatus.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ServoWgt</class>
   <extends>QWidget</extends>
   <header>servowgt.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
